plugins/uefi/meson.build

```diff
@@ -7,7 +7,18 @@ if efi_os_dir != ''
   cargs += '-DEFI_OS_DIR="' + efi_os_dir + '"'
 endif
 
-install_data(['uefi.quirk'],
+run_target('uefi-update-quirk',
+  command: [
+    python3.path(),
+    join_paths(meson.current_source_dir(), 'generate-quirk.py'),
+    '/var/lib/fwupd/remotes.d/lvfs/metadata.xml.gz', 'uefi-lvfs-generated.quirk'
```

Comment:
If we ever get to the point that releases can be just "Click a button on Github", then this probably needs to be able to `curl`/`wget` the URL directly.